pages/About Data Triad.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <div class="container">
+        <article>
```

Comment:
Probably remove the `<article>` since this is for a header tag. So something like this:

```html
<div>
  <h1 class:"title">About Data Triad</h1>
</div>